apps/air-discount-scheme/backend/src/app/modules/discount/discount.controller.ts

```diff
@@ -8,12 +16,15 @@ import {
 } from './dto'
 import { DiscountService } from './discount.service'
 import { NationalRegistryService } from '../nationalRegistry'
+import { FlightService } from '../flight'
 
 @Controller('api/private')
 export class PrivateDiscountController {
   constructor(
     private readonly discountService: DiscountService,
     private readonly nationalRegistryService: NationalRegistryService,
+    @Inject(forwardRef(() => FlightService))
```

Comment:
Import the module instead. Property based injection should only be used in very rare and specific cases